### ISSUE TITLE
Create camotics wikipedia page

### ISSUE THREAD

**Issue by luzpaz** [CauldronDevelopmentLLC/CAMotics#371]
Any volunteers to create a wikipedia page for this project?

**Comment by jacoblf** [comment]
Is this a real issue?

**Comment by luzpaz** [comment]
Yes, because it impacts repology stats: https://repology.org/project/camotics/versions (wikipedia is queried as well)
